src/components/OrderableList/useDraggableItem.tsx

```diff
@@ -0,0 +1,34 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
```

Comment:
I can not put another comment on the line 6 but could you change each `export function` to `export const ... = (...) => {}`

Reply:
fixed